taginfo.json

```diff
@@ -0,0 +1,41 @@
+{
+    "data_format": 1,
+    "data_url": "https://raw.githubusercontent.com/Rudloff/openvegemap/master/taginfo.json",
+    "project": {
+        "name": "OpenVegeMap",
+        "description": "Find vegetarian and vegan restaurants in your city",
+        "project_url": "https://github.com/Rudloff/openvegemap",
+        "doc_url": "https://play.google.com/store/apps/details?id=pro.rudloff.openvegemap",
+        "icon_url": "https://raw.githubusercontent.com/Rudloff/openvegemap/master/img/logo_32.png"
+    },
+    "tags": [
+{ "key": "diet:vegan",  "object_types": [ "node", "area" ], "description": "Displayed in popup" },
```

Comment:
I'm not sure I understand why you added these additional elements without any `value` key.

Reply:
The key and its value are shown in the popup regardless of its value. For example, [at this location](https://openvegemap.netlib.re/#zoom=19&lat=40.672229&lon=-73.957522), the two businesses west of Franklin Avenue display "limited" for their popup's Vegan field.

Reply:
Thanks for the explanation :slightly_smiling_face: